scripts/vmbuild.rb

```diff
@@ -134,6 +136,12 @@ def verify_run(output)
     params = "--parameters #{target_file}"
     $log.info "Running #{target} target_image #{imgfac_target} using parameters: #{params}"
 
+    if !vhd_image && !gce_image
+      $log.info "Running #{target} target_image with #{imgfac_target} and uuid: #{uuid}"
+      output = `./imagefactory --config #{IMGFAC_CONF} target_image --id #{uuid} #{imgfac_target}`
+      uuid   = verify_run(output)
+      $log.info "#{target} target_image with imgfac_target: #{imgfac_target} and uuid #{uuid} complete"
+
```

Comment:
@JPrause According to the bot, this now has a syntax error and can't work.  See the missing end for the if statement here.